packages/ci/src/lib/cli/commands/collect.ts

```diff
@@ -0,0 +1,29 @@
+import { executeProcess } from '@code-pushup/utils';
+import type { CommandContext } from '../context';
+import {
+  type PersistedCliFiles,
+  persistCliOptions,
+  persistedCliFiles,
+} from '../persist';
+
+export async function collect({
```

Comment:
maybe `runCollect` to avoid confusion with the `collect` function in `core` and the `collect` command in `cli`?

Reply:
Renamed all these command helpers.